src/feeds/FeedReader.ts

```diff
@@ -0,0 +1,144 @@
+import { MatrixClient } from "matrix-bot-sdk";
+import { BridgeConfigFeeds } from "../Config/Config";
+import { ConnectionManager } from "../ConnectionManager";
+import { FeedConnection } from "../Connections";
+import LogWrapper from "../LogWrapper";
+import { MessageQueue } from "../MessageQueue";
+
+import axios from "axios";
+import Parser from "rss-parser";
+
+const log = new LogWrapper("FeedReader");
+
+export class FeedError extends Error {
+    constructor(
+        public url: string,
+        public cause: Error,
+    ) {
+        super(`Error fetching feed ${url}: ${cause.message}`);
+    }
+}
+
+export interface FeedEntry {
+    feed: {
+        title: string,
+        url:   string,
+    },
+    title: string,
+    link:  string,
+}
+
+interface AccountData {
+    [url: string]: string[],
+}
+
+function stripHtml(input: string): string {
+    return input.replace(/<[^>]*?>/g, '');
```

Comment:
Heh, I'm surprised it's that easy. If it works it works though!

Reply:
I was hoping there's already something in yarn.lock that I won't have to reimplement, but there isn't and I don't feel like pulling in another dependency for something this stupid :P I *think* it covers all the bases. We may want to strip markdown to, just in case, since we appear to interpret it later when posting the message to the room.